api/BluetoothRemoteGATTService.json

```diff
@@ -269,102 +269,6 @@
           }
         }
       },
-      "getCharacteristics": {
```

Comment:
Running https://mdn-bcd-collector.appspot.com/tests/api/BluetoothRemoteGATTService/getCharacteristic in Chrome shows this to be supported.

@vinyldarkscratch would you be able to send update PRs for anything Web Bluetooth, to make it easier to review this PR?

Reply:
Can confirm this property is supported!